demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -2142,7 +2238,7 @@ export class DemoMeetingApp
           this.appendNewSpeakerTranscriptDiv(segment, speakerToTranscriptSpanMap);
         } else {
           const transcriptSpan = speakerToTranscriptSpanMap.get(newSpeakerId);
-          transcriptSpan.innerText = transcriptSpan.innerText + '\u00a0' + segment.content;
+          transcriptSpan.innerHTML = transcriptSpan.innerHTML + '\u00a0' + segment.content;
```

Comment:
Any reason we need to set `innerHTML` here instead?

Reply:
Same as below